acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.test.acceptance.dsl;
+
+import static org.apache.tuweni.io.file.Files.deleteRecursively;
+import static org.apache.tuweni.toml.Toml.tomlEscape;
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.artemis.util.Waiter.waitFor;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.TimeUnit;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.artemis.provider.JsonProvider;
+import tech.pegasys.artemis.test.acceptance.dsl.data.BeaconHead;
+
+public class ArtemisNode {
+  private static final Logger LOG = LogManager.getLogger();
+  private static final String ARTEMIS_BINARY =
+      new File(System.getProperty("artemis.binary.path", "../build/install/artemis/bin/artemis"))
+          .getAbsolutePath();
+
+  private final SimpleHttpClient httpClient;
+
+  private boolean started = false;
+  private File configFile;
+  private Process process;
+  private Path workDir;
+
+  ArtemisNode(final SimpleHttpClient httpClient) {
+    this.httpClient = httpClient;
+  }
+
+  public void start() throws Exception {
+    assertThat(started).isFalse();
+    started = true;
+    workDir = Files.createTempDirectory("artemis");
+    configFile = File.createTempFile("config", ".toml");
+    configFile.deleteOnExit();
+    new ConfigBuilder().writeTo(configFile);
+    LOG.info(
+        "Starting artemis from {} with config file {}",
+        ARTEMIS_BINARY,
+        configFile.getAbsolutePath());
+    process =
+        new ProcessBuilder(
+                ARTEMIS_BINARY, "--config", configFile.getAbsolutePath(), "--logging", "DEBUG")
```

Comment:
I don't think we should always turn logging to debug - maybe add an option to manually override the logging level?
```suggestion
                ARTEMIS_BINARY, "--config", configFile.getAbsolutePath())
```

Reply:
Yep meant to remove that.